SiaInterface/bin/Renter.dart

```diff
@@ -0,0 +1,55 @@
+import 'package:http/http.dart' as http;
+import 'dart:io';
+
+class Renter {
+
+   /*
+   * nickname:     file that has been uploaded to the network.
+   * destination:  the path that the file will be downloaded to.
+   */
+
+  download(String nickname, String destination) {
+    var url = "http://localhost:9980/renter/download";
+
+    http.post(url, body: 'name=doodle&color=blue').then((response) {
+       print("Response status: ${response.statusCode}");
```

Comment:
instead of printing, have this function take a callback that will give the response data

When you give the response data, put it all into a class so I know exactly what I'm getting